### ISSUE TITLE
@uuid@ not working as @integer@

### ISSUE THREAD

**Issue by Canadadry** [coduo/php-matcher#112]
Hi, 

I am validating url containing id as integer, and I used to do it like this (the following code print `match`.

```
if (!PHPMatcher::match("/test/0", "/test/@integer@", $error)) { 
    echo $error; // in case of error message is set on $error variable via reference
}
else
{
	echo "match\n";
}
```

I am switching to UUID and the same test case is not working. The following code print `Type pattern "@uuid@" is not supported by TextMatcher.`:

```
if (!PHPMatcher::match("/test/00200000-0000-4000-a000-000000000000", "/test/@uuid@", $error)) { 
    echo $error."\n"; // in case of error message is set on $error variable via reference
}
else
{
	echo "match\n";
}```

**Comment by Canadadry** [comment]
After further investigation it seems that you have forget to add uuid in the RegexConverter. 

I have just add this in the switch case 

```
            case 'uuid':
                return "([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})";
```

And now it is working as expected.

**Comment by thomasbisignani** [comment]
IMO this issue is solved with this PR #130 and can be closed. 
What do you think @norzechowicz ?
